src/sprite_battler.cpp

```diff
@@ -259,7 +259,7 @@ void Sprite_Battler::CreateSprite() {
 
 	SetX(battler->GetBattleX());
 	SetY(battler->GetBattleY());
-	SetZ(battler->GetBattleY()); // Not a typo
+	SetZ(PriorityBattle + battler->GetBattleY()); // Not a typo
```

Comment:
While we are at it, why not making this comment a bit more constructive?
Right as it is, this needs thinking over the purpose. Maybe something like this:
`// overlaps from top to bottom`